roles/openstack-stack/templates/heat_stack.yaml.j2

```diff
@@ -162,13 +162,8 @@ resources:
           port_range_max: 22
           remote_ip_prefix: {{ ssh_ingress_cidr }}
         - direction: ingress
-          protocol: tcp
-          port_range_min: 53
-          port_range_max: 53
-        - direction: ingress
-          protocol: udp
-          port_range_min: 53
-          port_range_max: 53
+          protocol: icmp
+          remote_ip_prefix: {{ ssh_ingress_cidr }}
```

Comment:
Can we add another rule that makes the hosts pingable within the network as well (i.e. master can ping the infra, etc.). We do a similar thing for the DNS rules.